app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java

```diff
@@ -1350,9 +1349,12 @@ private void onFunctionKey(final int primaryCode, final Key key, final int multi
                 if (mVoiceRecognitionTrigger.isInstalled()) {
                     mVoiceRecognitionTrigger.startVoiceRecognition(getCurrentAlphabetKeyboard().getDefaultDictionaryLocale());
                 } else {
-                    Intent voiceInputNotInstalledIntent = new Intent(getApplicationContext(), VoiceInputNotInstalledActivity.class);
-                    voiceInputNotInstalledIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                    startActivity(voiceInputNotInstalledIntent);
+                    Context context = getApplicationContext();
```

Comment:
Not all users have Play Store installed. This information could be contained in README, Help, whatever. But I agree that the Toast should mention that the information is in README